config/config.go

```diff
@@ -76,9 +78,8 @@ type API struct {
 }
 
 type Transport struct {
-	Enabled                bool   `toml:"enabled"`
-	Workers                int    `toml:"workers"`
-	SleepSeconds           int    `toml:"sleep_seconds"`
-	DataDir                string `toml:"data"`
-	FreeSpaceRequiredBytes int64  `toml:"free_space_required_bytes"`
+	Queue   string `toml:"queue"`
+	Storage string `toml:"storage"`
+	DataDir string `toml:"data"`
```

Comment:
I added `DataDir` and `Workers` since they seem to be required by `queuestorage` (when ran it shows an error `mkdir : no such file or directory`).